clio-server/src/test/scala/org/broadinstitute/clio/server/service/ServerServiceSpec.scala

```diff
@@ -132,29 +134,24 @@ class ServerServiceSpec
       Some(URI.create("/mock")),
       Some(1L)
```

Comment:
We should also add a test with `mockDefaultField` set to something other than `"mock-default"`, and check that it doesn't get overwritten by recovery.

Reply:
Done